stdlib/toolchain/Compatibility56/CompatibilityOverrideConcurrency.def

```diff
@@ -343,10 +343,6 @@ OVERRIDE_TASK_STATUS(task_escalate, JobPriority,
                      swift::, (AsyncTask *task, JobPriority newPriority),
                      (task, newPriority))
 
-OVERRIDE_TASK_STATUS(task_getNearestDeadline, NearestTaskDeadline,
```

Comment:
I think this might need to stay in order to maintain the layout of the overrides struct.

Reply:
Ah cause we already have the override hook present even though it is not actually called/used anywhere?

Reply:
Exactly. We magick up a struct containing a function pointer field for each of these things, and we want that struct to have the same definition in the compatibility library and in the runtime it applies to.

Reply:
@rjmccall added really solid documentation on what we can and cannot do to the compatibility overrides here to avoid breaking the ABI; https://github.com/apple/swift/blob/main/stdlib/public/CompatibilityOverride/CompatibilityOverrideRuntime.def